includes/wc-core-functions.php

```diff
@@ -812,6 +812,18 @@ function wc_get_log_file_path( $handle ) {
 }
 
 /**
+ * Get a log file name.
+ *
+ * @since 3.3
+ *
+ * @param string $handle Name.
+ * @return string The log file name.
+ */
+function wc_get_log_file_name( $handle ) {
```

Comment:
Since this only works with file based logs, perhaps a function is not needed? Depends on the action taken on my other comments I guess.

Reply:
The function just above this is also just for file logging, worth keeping around in case you want to easily get a filename?